pages/windows/reg-query.md

```diff
@@ -0,0 +1,23 @@
+# reg query
+
+> Display the values of keys and sub keys in the registry.
+
+- Display all values of a key:
+
+`reg query {{key_name}}`
+
+- Display a specific value of a key:
+
+`reg query {{key_name}} /v {{value}}`
+
+- Display a specific value of a key and recurse into sub keys:
```

Comment:
You don't specify a value here, yet the description says "a specific value of a key". Typo ? Does this sound better - `Display all values of a key and its sub keys` ?

Reply:
Yes. Sorry, that is a better description. 👍